CHANGES_NEXT_RELEASE

```diff
@@ -1,7 +1,13 @@
+* [FEATURE] STH connection to MongoDB replica sets (#34)
```

Comment:
Ups! Fixed in https://github.com/telefonicaid/IoT-STH/commit/093fcc646a2622644521353cce3090835fb28c2d